po/nl.po

```diff
@@ -1827,16 +1824,16 @@ msgstr "Migratie is afgebroken omdat mkdir mislukte voor map `%s': %s"
 #. The "%{}" thing is a number, a zero-based offset into a string.
 #: rust/libnewsboat/src/filterparser.rs:263
 msgid "Parse error: trailing characters after position %{}: %s"
-msgstr ""
+msgstr "Fout tijdens ontleding: onverwachte tekens na positie %{}: %s"
 
 #. The "%{}" thing is a number, a zero-based offset into a string.
 #: rust/libnewsboat/src/filterparser.rs:270
 msgid "Parse error at position %{}: expected %s"
-msgstr ""
+msgstr "Fout tijdens ontleding op positie %{}: verwachtte %s"
```

Comment:
It looks like `%s` will be replaced with one of the following strings, which are not being translated:
https://github.com/newsboat/newsboat/blob/ba0bdb31860773e1161064635ff3cace87f97cbb/rust/libnewsboat/src/filterparser.rs#L64-L66

I don't have time at the moment to fix this myself.
Is it worth opening an issue for this?

Reply:
Oh, good catch! I simply assumed that they're already translated when I updated my translations ._.

No need for an issue; I'll fix those lines in a PR in a few minutes. Since Rust FilterParser is not used at the moment, all these strings are dead code, so there's no need to update POT and translations to include them — that can wait until after the release.

Reply:
Fixed by #1191.